utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -19,6 +19,22 @@
 
```

Comment:
Just wondering, why all these rearrangements of the position of _import_ lines?
This is done on couple of files if I see it correctly?

Reply:
I uncheckd the "optimize import" feature of IDEA, hope that helps.